sdk/tests/deqp/functional/gles3/es3fTransformFeedbackTests.js

```diff
@@ -939,8 +938,6 @@ goog.scope(function() {
 
         this.m_transformFeedback = gl.createTransformFeedback();
 
-        es3fTransformFeedbackTests.GLU_EXPECT_NO_ERROR(gl, gl.getError(), 'es3fTransformFeedbackTests.init');
```

Comment:
Why are the assertions around glGetError returning NO_ERROR being removed? Is it because browsers are actually generating errors? If so, then we should fix the browsers and leave the assertions in place.